src/chromo/models/sibyll.py

```diff
@@ -137,34 +141,80 @@ class Sibyll23(SIBYLLRun):
 
 class Sibyll23c(SIBYLLRun):
     _version = "2.3c"
+    _projectiles = standard_projectiles | {
+        3112,
+        3122,
+        3312,
+        3322,
+        3222,
+        411,
+        421,
+        4232,
+        431,
+        4122,
+        4132,
+        4232,
+        431,
+        4332,
+    }
```

Comment:
Likewise, shouldn't this be in SIBYLLRun?

Reply:
Ok